### ISSUE TITLE
Changes to submodule take no immediate effect in celeryworker

### ISSUE THREAD

**Issue by jotelha** [ContactEngineering/topobank-stack-development#9]
I can make changes to submodules (plugins) on the frontend side and test the effect immediately when reloading the webpage in the browser.

When I make changes to the backend part (analysis functions running on celeryworker), triggering a rerun is not enough. Apparently I need to restart the worker service `docker compose restart celeryworker` and trigger a rerun of the analysis for the changes to take effect.

**Comment by mcrot** [comment]
Yes, I know this. The IDE only takes care for restarting the django runserver process in the "django" container,
but not in the celery worker. I've also restarted the celeryworker manually in the past for all changes to the analysis function. If you find a solution for that it would be great.
